services/kuber/server/longhorn/longhorn.go

```diff
@@ -175,7 +182,7 @@ func (l *Longhorn) deleteOldStorageClasses(existing, applied []string, kc kubect
 		//if not found in applied, delete the sc
 		if !found {
 			err := kc.KubectlDeleteResource("sc", ex, "")
-			log.Info().Msgf("Deleting storage class %s", ex)
+			log.Debug().Msgf("Deleting storage class %s", ex)
```

Comment:
I think that log about removing a StorageClass would be usefull for user, not only in Debug mode. WDYT?